src/base/iana/rcode.rs

```diff
@@ -190,7 +190,7 @@ impl Rcode {
         }
     }
 
-    /// Returns the integer value for this rcode.
+    /// Returns the 4-bit integer value for this rcode.
```

Comment:
Would it perhaps be better to keep the old text and add a line below saying that “Only the lower 4 bits of the returned octet are used by the rcode. The upper four bits are always zero.”?

Reply:
Is that better?

Reply:
Yes to both ;)